src/Command/Module/DownloadCommand.php

```diff
@@ -185,7 +185,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
                         return 1;
                     } else {
                         $version = $io->choice(
-                            $this->trans('commands.site.new.questions.composer-release'),
+                            sprintf(
+																$this->trans(
+																		'commands.site.new.questions.composer-release'),
```

Comment:
@enzolutions these are the 4 lines changes.
this PR only affects to this file `src/Command/Module/DownloadCommand.php `